lib/drivers/animasDriver.js

```diff
@@ -1759,7 +1760,7 @@ module.exports = function (config) {
         postrecords.push(bgRecord);
       }
       else {
-        console.log('Discarding control solution test');
+        debug('Discarding control solution test');
```

Comment:
Should some other of these `debug` statements be wrapped in a `if (verbose) {}` clause?